src/server/constants/prometheus.ts

```diff
@@ -0,0 +1,14 @@
+import { MetricsDefinition } from '../../types';
+
+export const metrics: MetricsDefinition = {
+  data: {
+    balance: 0,
```

Comment:
I don't think this is a good idea to initialize the balance with 0.
It will lead to prometheus potentially reporting this and trigger alerts about the balance being 0 whereas it currently is not empty.

Reply:
Ah this was simply copied from what it already is, I'm not too confident in what else this should be? Might need @AndreiEres to weigh in here

Reply:
The current code skips retuning the metrics at all until a real value is known (it does not return 0 unless it is really 0).
Not sure what Prometheus' defaults are but `null` or `undefined` would be better.

Reply:
I would disagree with that, becaue if we merge this PR with the balance initializing at 0, we will get false alarms about the balance being too low. We do not have the issue today so the PR in the current state would introduce a regression. 

We don't really need another PR to fix regressions introduced in a PR if we are aware of them.

Reply:
> We do not have the issue today so the PR in the current state would introduce a regression.

I don't understand that considering this is what the existing code already does? Are you saying we haven't deployed this code yet?